packages/@atjson/source-gdocs-paste/public/app.js

```diff
@@ -1,17 +1,18 @@
 import GoogleDocsPasteSource from '../src';
 import * as fs from 'fs';
+import OffsetSource from '@atjson/offset-annotations';
```

Comment:
Is this necessary for this change? Might be better to bundle this with a separate PR? (I'm fine either way, just raising in case this accidentally snuck in)

Reply:
Oops, that was my mistake!